packages/honeycomb-opentelemetry-web/src/util.ts

```diff
@@ -71,16 +75,52 @@ export const getTracesEndpoint = (options?: HoneycombOptions) => {
 
   // use `endpoint` option if provided and append '/v1/traces' if not already appended
   if (options?.endpoint) {
-    return maybeAppendTracesPath(options.endpoint);
+    return maybeAppendPath(options.endpoint, TRACES_PATH);
   }
 
   return DEFAULT_TRACES_ENDPOINT;
 };
 
+export const getMetricsEndpoint = (options?: HoneycombOptions) => {
```

Comment:
Can we add unit tests for this similar to `getTracesEndpoint`? Even though its a bit tedious it really does help to keep the coverage high for when we make changes to these functions in the future.